package.json

```diff
@@ -75,68 +76,48 @@
     "react-native-storybook-loader": "^2.0.4",
     "react-native-swiper": "^1.6.0",
     "react-native-tab-view": "^2.15.2",
-    "react-native-vector-icons": "^7.1.0",
+    "react-native-vector-icons": "^8.1.0",
     "react-native-web": "^0.14.10",
     "react-native-webview": "^11.0.2",
     "react-native-youtube-iframe": "^1.4.0",
+    "react-query": "^3.19.0",
     "rn-fetch-blob": "^0.12.0",
     "styled-components": "^5.2.1",
     "validator": "^13.5.2",
-    "yarn": "^1.22.10"
+    "yarn": "^1.22.17"
```

Comment:
Do we really need yarn as a dependency?

Reply:
I don't think so, even for CI it uses the global version I think.

Reply:
Will remove it